src/api/app/controllers/webui/package_controller.rb

```diff
@@ -471,10 +471,10 @@ def rpmlint_result
   end
 
   def rpmlint_log
-    rpmlint_log_file = Backend::Api::BuildResults::Binaries.rpmlint_log(params[:project], params[:package], params[:repository], params[:architecture])
+    rpmlint_log_file = RpmlintLogExtractor.new(params.slice(:project, :package, :repository, :architecture).permit!).call
```

Comment:
It's a Rails convention to have the strong params declaration in a private method, we should follow that.

Reply:
True. I created the `rpmlint_log_params` method. Please take a look again.